server/chat-plugins/blackjack.ts

```diff
@@ -663,10 +663,12 @@ export const commands: ChatCommands = {
 			if (!this.can('minigame', null, room)) return;
 			if (room.game) return this.errorReply("There is already a game running in this room.");
 			if (room.blackjackDisabled) return this.errorReply("Blackjack is currently disabled in this room.");
+			if (target && isNaN(parseFloat(target))) {
```

Comment:
```suggestion
			const autostartTime = target ? parseFloat(target) : 0;
			if (isNaN(autostartTime)) {
```

You should also use `autostartTime` instead of `target` in the `Blackjack` constructor.